File: class14/cpp/AP1401-Exemplo01.cpp
#include <iostream>
int main() {
    double a, b, c, maior;
    std::cout << "Número 1: "; std::cin >> a;
    std::cout << "Número 2: "; std::cin >> b;
    std::cout << "Número 3: "; std::cin >> c;
    if (a >= b && a >= c) maior = a;
    else if (b >= c) maior = b;
    else maior = c;
    std::cout << "O maior número é " << maior << std::endl;
    return 0;
}

File: class14/java/AP1401-Exemplo01.java
import java.util.Scanner;
public class MaiorDeTres {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Número 1: "); double a = sc.nextDouble();
        System.out.print("Número 2: "); double b = sc.nextDouble();
        System.out.print("Número 3: "); double c = sc.nextDouble();
        double maior;
        if (a >= b && a >= c) maior = a;
        else if (b >= c) maior = b;
        else maior = c;
        System.out.println("O maior número é " + maior);
        sc.close();
    }
}